core/src/autogluon/core/scheduler/scheduler.py

```diff
@@ -130,6 +140,24 @@ def _release_resource_callback(fut):
         job.add_done_callback(_release_resource_callback)
         return job
 
+    @staticmethod
+    def _worker(pickled_fn, return_list, gpu_ids, args):
+        """Worker function in thec client
+        """
+        fn = dill.loads(pickled_fn)
```

Comment:
Can ypu comment on why one has to serialize/deserialize fn here, but not in the old code?

Reply:
Added.